generators/app/templates/src/app/_app.component.ts

```diff
@@ -119,7 +113,7 @@ export class AppComponent implements OnInit {
 
   private onCordovaReady() {
     if (window['cordova']) {
-      this.keyboard.hideKeyboardAccessoryBar(true);
+      Keyboard.hideFormAccessoryBar(true);
```

Comment:
use `window['Keyboard']` to avoid the `declare var` on top.

Not pretty, but `@ionic-native/keyboard` should be updated soon anyways.

Reply:
donezo :)